PKGBUILD

```diff
@@ -10,7 +10,6 @@ license=('GPL')
 depends=('qt5-base')
 makedepends=('git' 'gcc')
 provides=('zenity')
```

Comment:
Doesn't, does it?
https://github.com/luebking/qarma/commit/605ea4213406718ba869dd146875195e57488786

Reply:
> Doesn't, does it?

With add of "ln -s /usr/bin/qarma "$pkgdir/usr/bin/zenity"" it conflicts

By not providing /usr/bin/zenity isn't correct to call it a drop-in replacement

Reply:
Depends on who drops it in - the user into their packages or the developer into their code.
In any event we can't say that it provides zenity when it actually doesn't.

That's why there's https://github.com/luebking/qarma/blob/master/qarma-zenity/PKGBUILD
Otherwise I could just name the binary "zenity"

Reply:
Ok, I missed qarma-zenity pkgbuild.

provides=('zenity') from qarma pkgbuild can be removed then